nautobot/utilities/templatetags/helpers.py

```diff
@@ -212,41 +306,81 @@ def has_one_or_more_perms(user, permissions_list):
     return False
 
 
+@library.filter()
 @register.filter()
 def split(string, sep=","):
-    """
-    Split a string by the given value (default: comma)
+    """Split a string by the given value (default: comma)
+
+    Args:
+        string (str): string to split into a list
+        sep (str default=,): separator to look for in the string
+
+    Returns:
+        [list]: List of string, if the separator wasn't found, list of 1
     """
     return string.split(sep)
 
 
+@library.filter()
 @register.filter()
 def as_range(n):
-    """
-    Return a range of n items.
+    """Return a range of n items.
+
+    Args:
+        n (int, str): Number of element in the range
+
+    Returns:
+        [list, Range]: range function from o to the value provided. Returns an empty list if n is not valid.
+
+    Example:
+        {% for i in record.parents|as_range %}
+            <i class="mdi mdi-circle-small"></i>
+        {% endfor %}
     """
     try:
         int(n)
-    except TypeError:
+    except (TypeError, ValueError):
```

Comment:
When's a `ValueError` seen here?

Reply:
if n is a string that can't be converted into an int
```
>>> int("one")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'one'
```